startup.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
```

Comment:
should this be called sync_schedules.py? am a bit confused with things now.

Reply:
(there is also run.sh which is an entry point - maybe we need to consolidate things a bit, but ideally not with a hard dependency on python for things that don't need python?)